### ISSUE TITLE
Fails to escape filenames with spaces

### ISSUE THREAD

**Issue by ed-george** [robertoestivill/densinator#2]
The script fails to correctly process files that contain spaces in the name

e.g. for `hello world.png`

```
Scaling hello to 100% in generated/drawable-xxhdpi/hello
convert: unable to open image `hello': No such file or directory @ error/blob.c/OpenBlob/2695.
convert: no decode delegate for this image format `' @ error/constitute.c/ReadImage/501.
convert: no images defined `generated/drawable-xxhdpi/hello' @ error/convert.c/ConvertImageCommand/3230.
...
Scaling world.png to 100% in generated/drawable-xxhdpi/world.png
convert: unable to open image `world.png': No such file or directory @ error/blob.c/OpenBlob/2695.
convert: unable to open file `world.png' @ error/png.c/ReadPNGImage/3913.
convert: no images defined `generated/drawable-xxhdpi/world.png' @ error/convert.c/ConvertImageCommand/3230.
```

What would be helpful as a feature that could extend this fix, is the replacement of spaces with underscores in the output files.

**Comment by robertoestivill** [comment]
You are right. I will find a way to get around this limitation. 
Also, some transformation will need to happen (replace space with '_' ), as android resources can not have empty spaces in the file name.
Thanks for reporting :)

**Comment by robertoestivill** [comment]
Should be fixed by d7554fca927c8a2f65a6c9c56c70301f4f3b1c57

Thanks
